src/launchpad/artifact_processor.py

```diff
@@ -459,6 +459,9 @@ def _get_artifact_type(artifact: Artifact) -> ArtifactType:
             apple_app_info=apple_app_info,
             android_app_info=android_app_info,
             dequeued_at=dequeued_at,
+            cli_version=app_info.cli_version,
+            fastlane_plugin_version=app_info.fastlane_plugin_version,
```

Comment:
Not sure I see a point of setting these here if you're already setting them in the `apple_app_info`/`android_app_info` - I'd recommend only one spot or the other

Reply:
`apple_app_info` and `android_app_info` are already top-level fields in the `UpdateData` model. You also have reference to those in the backend. In this case, you're still duplicating the fields, `cli_version`, `fastlane_plugin_version` and `gradle_plugin_version`. Those can and should be pulled from only one spot or another (top-level in UpdateData or in apple/android_app_info), not duplicated.

IMO there should be a PR here to update the backend handling to get those values from the android/apple_app_info fields and we should remove the top-level fields from UpdateData. Simpler and leverages the app_info models that are already being passed, and we don't duplicate data.